ui.frontend/src/main/webpack/site/js/components/searchInput.ts

```diff
@@ -175,13 +175,13 @@ const debouncedSearch = (autoSuggestionDebounceTime: number) =>
   )
 
 const buildSearchInput = ({
-  id,
-  searchFieldPlaceholderText,
-  autocompleteUrl,
-  autocompleteTriggerThreshold,
-  autoSuggestionDebounceTime = 500,
-  searchContainer,
-}: SearchInputOptions): HTMLInputElement => {
+                            id,
+                            searchFieldPlaceholderText,
+                            autoCompleteUrl,
+                            autoCompleteTriggerThreshold: autoCompleteTriggerThreshold,
```

Comment:
You can keep autoCompleteTriggerThreshold with the ": autoCompleteTriggerThreshold" it is the same, just shorter.